guides/akka-cluster-kubernetes-k8s-deploy/akka-cluster-example/build.sbt

```diff
@@ -0,0 +1,74 @@
+import de.heikoseeberger.sbtheader.HeaderPattern
+import scalariform.formatter.preferences.AlignSingleLineCaseStatements
+import com.typesafe.sbt.packager.docker._
+
+lazy val akkaVersion = "2.5.4"
+
+name := "myapp"
+
+scalaVersion := "2.11.11"
+
+enablePlugins(AutomateHeaderPlugin, JavaServerAppPackaging)
+
+resolvers += Resolver.bintrayRepo("hseeberger", "maven")
+
+libraryDependencies ++= List(
+  "com.typesafe.akka" %% "akka-actor"          % akkaVersion,
+  "com.typesafe.akka" %% "akka-cluster"        % akkaVersion,
+  "com.typesafe.akka" %% "akka-http"           % "10.0.10",
+  "de.heikoseeberger" %% "akka-http-play-json" % "1.19.0-M2",
+  "com.typesafe.play" %% "play-json"           % "2.6.5",
+  "org.scalatest"     %% "scalatest"           % "3.0.1"     % "test",
+  "com.typesafe.akka" %% "akka-testkit"        % akkaVersion % "test"
+)
+
+dockerEntrypoint ++= Seq(
+  """-DakkaActorSystemName="$AKKA_ACTOR_SYSTEM_NAME"""",
+  """-Dakka.remote.netty.tcp.hostname="$(eval "echo $AKKA_REMOTING_BIND_HOST")"""",
+  """-Dakka.remote.netty.tcp.port="$AKKA_REMOTING_BIND_PORT"""",
+  """-Dakka.cluster.seed-nodes.0="akka.tcp://${AKKA_ACTOR_SYSTEM_NAME}@${AKKA_SEED_NODE_HOST_0}:${AKKA_SEED_NODE_PORT}"""",
+  """-Dakka.cluster.seed-nodes.1="akka.tcp://${AKKA_ACTOR_SYSTEM_NAME}@${AKKA_SEED_NODE_HOST_1}:${AKKA_SEED_NODE_PORT}"""",
+  """-Dakka.cluster.seed-nodes.2="akka.tcp://${AKKA_ACTOR_SYSTEM_NAME}@${AKKA_SEED_NODE_HOST_2}:${AKKA_SEED_NODE_PORT}"""",
+  "-Dakka.io.dns.resolver=async-dns",
+  "-Dakka.io.dns.async-dns.resolve-srv=true",
+  "-Dakka.io.dns.async-dns.resolv-conf=on",
+  """-DhttpHost="$HTTP_HOST"""",
+  """-DhttpPort="$HTTP_PORT"""",
+  """-DclusterMembershipAskTimeout="$CLUSTER_MEMBERSHIP_ASK_TIMEOUT""""
+)
+dockerCommands :=
+  dockerCommands.value.flatMap {
+    case ExecCmd("ENTRYPOINT", args @ _*) => Seq(Cmd("ENTRYPOINT", args.mkString(" ")))
+    case v => Seq(v)
+  }
+dockerRepository := Some("mygroup")
+dockerUpdateLatest := true
+dockerBaseImage := "local/openjdk-jre-8-bash"
+
+ScalariformKeys.preferences :=
+  ScalariformKeys.preferences.value
+    .setPreference(AlignSingleLineCaseStatements, true)
+    .setPreference(AlignSingleLineCaseStatements.MaxArrowIndent, 100)
+
+HeaderPlugin.autoImport.headers := Map(
+  "scala" -> (
+    HeaderPattern.cStyleBlockComment,
+    """|/*
+       | * Copyright © 2014-2016 Lightbend, Inc. All rights reserved.
+       | * No information contained herein may be reproduced or transmitted in any form
+       | * or by any means without the express written permission of Lightbend, Inc.
+       | */
+       |
+       |""".stripMargin
+    ),
+  "conf" -> (
+    HeaderPattern.hashLineComment,
+    """|# Copyright © 2014-2016 Lightbend, Inc. All rights reserved.
+       |# No information contained herein may be reproduced or transmitted in any form
+       |# or by any means without the express written permission of Lightbend, Inc.
+       |
+       |""".stripMargin
+    )
+)
```

Comment:
Since this is an open source repo, I don't think this header is appropriate. If you use the latest `sbt-header` plugin, this config should be enough:

https://github.com/typesafehub/reactive-cli/blob/master/build.sbt#L15-L19

Reply:
Oops... thanks for the catch!

Reply:
Header files now updated as part of SBT update to `1.0.2`.